_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -100,55 +100,55 @@ Autocomplete term, only valid for textarea.
 Two autocomplete terms.
 
 ```html
-<input autocomplete="Work EMail" />
+<label>Work email<input autocomplete="Work EMail"/></label>
```

Comment:
Does this autocomplete value reference something? Its unusual camel casing in my experience, so doesn't read like a variable name, or something legit for an autocomplete value.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
autocomplete tokens are match case-insensitive, so this is technically OK.
I've nonetheless corrected it as it is indeed a bit confusing and definitely not what the example illustrates.

Reply:
Why is it not all lower case like the other examples of autocomplete?

Reply:
That, I don't know and deem it an unrelated issue that need (or not) to be tackled in another PR…